Tema6/src/main/java/SetString.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class SetString {
+
+    Set<String> setRandom = new HashSet<>();
+
+    public String randomString() {
+        final String alphaCharacters = "abcdefghijklmnopqrstuvwxyz";
+
+        String randomString1 = "";
+
+
+        Random rand = new Random();
+        int length = rand.nextInt(alphaCharacters.length());
+        char[] text = new char[length];
+
+        for (int i = 0; i < length; i++) {
+            text[i] = alphaCharacters.charAt(rand.nextInt(alphaCharacters.length()));
+
+        }
+        for (int i = 0; i < text.length; i++) {
+            randomString1 += text[i];
+        }
+
+
+        return randomString1;
+
+    }
+
+    public void addInSet(String randomString) {
+
+
+        setRandom.add(randomString);
```

Comment:
the logic here is a bit off, you don't need a parameter here, you can simply call the method to add to Set and compute the randomString inside the method, do you understand ? (don't complicate it)